rest/rest-sqs-testing-amazon-java-sdk/src/test/scala/org/elasticmq/rest/sqs/AmazonJavaSdkTestSuite.scala

```diff
@@ -298,7 +299,9 @@ class AmazonJavaSdkTestSuite extends FunSuite with Matchers with BeforeAndAfter
   test("should block message for the visibility timeout duration") {
     // Given
     val queueUrl = client.createQueue(new CreateQueueRequest("testQueue1")
-      .withAttributes(Map(defaultVisibilityTimeoutAttribute -> "1"))).getQueueUrl
+      .withAttributes(
+        Map(defaultVisibilityTimeoutAttribute -> "1", defaultMaxReceiveCountAttribute -> "3")))
```

Comment:
Does this need to be specified in every test? Shouldn't an undefined value work, meaning no DLQ support?